lib/modules/platform/codecommit/index.spec.ts

```diff
@@ -190,6 +197,32 @@ describe('modules/platform/codecommit/index', () => {
         )
       ).toBe('https://git-codecommit.eu-central-1.amazonaws.com/v1/repos/name');
     });
+
+    it('gets url with username and token', () => {
+      process.env.AWS_ACCESS_KEY_ID = 'access-key-id';
+      process.env.AWS_SECRET_ACCESS_KEY = 'secret-access-key';
+      process.env.AWS_REGION = 'eu-central-1';
+      process.env.AWS_SESSION_TOKEN = '';
+      const signer = new aws4.RequestSigner({
```

Comment:
is this fully offline?

Reply:
It says signs and prepares nodejs request for using aws signature so, I think not.
Ref: [github/aws4](https://github.com/mhart/aws4) [awsDocs](https://docs.aws.amazon.com/IAM/latest/UserGuide/create-signed-request.html)